src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -579,6 +579,18 @@ private function evaluate( $directive_value ) {
 		$path_segments = explode( '.', $path );
 		$current       = $store;
 		foreach ( $path_segments as $path_segment ) {
+			// Special case for numeric arrays to add a length property like JavaScript arrays.
```

Comment:
Should we include a note about the version when it was introduced? 
```suggestion
			/*
			 * Special case for numeric arrays to add a length property like JavaScript arrays.
			 * @since 6.8.0
			 */			 
```